CalcualtorLogic.js

```diff
@@ -171,14 +171,15 @@ class Calculator {
         this.updateResultElement();
     }
     evaluate() {
-        if (this.lastValue && this.currentResult !== null) {
+        if (this.lastValue && this.currentResult !== null && this.operator !== null) {
             let firstOperand = parseFloat(this.lastValue);
             let secondOperand = parseFloat(this.currentResult);
             let _result;
             if (this.operator === "+") _result = this.calculatorLogic.add(firstOperand, secondOperand);
             if (this.operator === "-") _result = this.calculatorLogic.subtract(firstOperand, secondOperand);
             if (this.operator === "*") _result = this.calculatorLogic.multiply(firstOperand, secondOperand);
             if (this.operator === "/") _result = this.calculatorLogic.divide(firstOperand, secondOperand);
+            if (this.lastValue !== null) this.lastValue === null;
```

Comment:
There's no reason for adding this if statement 
also, using `===` is wrong here

` this.lastValue = null ` is enough